src/util.js

```diff
@@ -29,22 +29,14 @@ exports.getStorageConfiguration = function (config, env) {
     return _.extend({}, common, config[env]);
 };
 
-// this method has been taken from cheerio module code
-// https://github.com/cheeriojs/cheerio/blob/master/lib/cheerio.js#L122
 /**
  * Checks if given string is html string
  * @param {String} str (can be markdown or html string)
  * @returns {boolean}
  */
 exports.isHtml = function (str) {
```

Comment:
what is the purpose of this method?
actually any MD can contain HTML tags and any HTML may start and end with just a string (`blah <strong>ololo</strong> blah`)